types/ldapjs/index.d.ts

```diff
@@ -1,88 +1,81 @@
-// Type definitions for ldapjs
+// Type definitions for ldapjs 1.0
 // Project: http://ldapjs.org
 // Definitions by: Charles Villemure <https://github.com/cvillemure>, Peter Kooijmans <https://github.com/peterkooijmans>, Pablo Moleri <https://github.com/pmoleri>, Michael Scott-Nelson <https://github.com/mscottnelson>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-
 /// <reference types="node" />
 
-import { EventEmitter } from "events";
+import { EventEmitter } from 'events';
+
+export class LDAPError extends Error {
+    name: 'LDAPError' | string;
```

Comment:
`'LDAPError' | string` is the same type as `string`, `80 | number` is the same type as `number`, and `any | LDAPError` is the same thing as `any`. In short, having the narrower type in the union isn't actually adding anything from a type checking perspective. To prevent confusion, it's best to simply use `string`, `number` and `any`.